xml2js/processors.d.ts

```diff
@@ -0,0 +1,11 @@
+declare module 'xml2js/lib/processors' {
+    export function firstCharLowerCase(name: string): string;
+
+    export function normalize(name: string): string;
+
+    export function parseBooleans(name: string): string;
```

Comment:
You are correct. It also means I need to change the signatures of the processor arrays in options to be `[(name: string) => any]`. Thanks.